src/Domain/Differ.php

```diff
@@ -30,7 +30,7 @@ public function __construct()
     /**
      * {@inheritdoc}
      */
-    public function diff($old, $new): string
+    public function diff(string $old, string $new, ?\SplFileInfo $file = null): string
     {
         return $this->differ->diff($old, $new);
```

Comment:
```suggestion
        return $this->differ->diff($old, $new, $file);
```

Reply:
@Jibbarth It seems like they have changed the signature of the differ. So, if we're gonna use the third parameter (`$file`), we need to pass one parameter from the `LongestCommonSubsequenceCalculator` type. WDYT?